source/command/analyzeCommand.ml

```diff
@@ -315,21 +335,26 @@ let run_analyze analyze_configuration =
     =
     AnalyzeConfiguration.analysis_configuration_of analyze_configuration
   in
-  Server.BuildSystem.with_build_system source_paths ~f:(fun build_system ->
-      Scheduler.with_scheduler
-        ~configuration:analysis_configuration
-        ~should_log_exception:(function
-          | Taint.TaintConfiguration.TaintConfigurationError _ -> false
-          | Taint.ModelVerificationError.ModelVerificationErrors _ -> false
-          | _ -> true)
-        ~f:(fun scheduler ->
-          with_performance_tracking ~debug ~f:(fun () ->
-              TaintAnalysis.run_taint_analysis
-                ~static_analysis_configuration
-                ~build_system
-                ~scheduler
-                ());
-          Lwt.return (ExitStatus.CheckStatus CheckCommand.ExitStatus.Ok)))
+  if static_analysis_configuration.verify_taint_config_only then (
+    with_performance_tracking ~debug ~f:(fun () ->
+        verify_taint_configuration ~static_analysis_configuration () |> ignore);
```

Comment:
please avoid `ignore` since it can be error prone.

For instance, it can lead to the function not actually being called when an argument is missing:
```
let foo x y = x + y

ignore (foo 1) # compiles with no warning
```

I would use `let (_ : type) = f () in ()` instead

Reply:
Ah thanks for that! I will keep this in mind :)